frontend/src/concepts/pipelines/content/configurePipelinesServer/PipelineDropdown.tsx

```diff
@@ -76,10 +76,19 @@ export const PipelineDropdown = ({
   };
   return (
     <Dropdown
-      data-testid="select-data-connection"
-      menuAppendTo="parent"
-      position={DropdownPosition.right}
-      toggle={<MenuToggle onClick={onToggle} icon={<KeyIcon />} />}
+      onOpenChange={(isOpened) => setIsOpen(isOpened)}
+      popperProps={{ position: 'right' }}
+      toggle={(toggleRef) => (
+        <MenuToggle
+          aria-label="Pipeline dropdown tooggle"
```

Comment:
This label doesn't make sense as this dropdown is used here. Previously there was no label.
![image](https://github.com/opendatahub-io/odh-dashboard/assets/14068621/bf15b97b-73b8-4913-809d-fb0b6d99e913)